docs/client/error_handling.md

```diff
@@ -0,0 +1,62 @@
+# <img align="center" src="../images/logo.png">  Error Handling
+
+## General
+
+Our generated clients raise exceptions defined in [`azure-core`][azure_core_exceptions]. While the base for all exceptions is [`AzureError`][azure_error],
+[`HttpResponseError`][http_response_error] is also a common base catch-all for exceptions, as these errors are thrown in the case of a request being made, and a non-successful
+status code being received from the service.
```

Comment:
is it worth linking here where we have descriptions of all azure.core exceptions? https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/core/azure-core#azure-core-library-exceptions

Reply:
def shoud've linked, thanks!